src/HotChocolate/PersistedQueries/src/PersistedQueries.Redis/Extensions/HotChocolateRedisPersistedQueriesServiceCollectionExtensions.cs

```diff
@@ -69,7 +75,7 @@ public static class HotChocolateRedisPersistedQueriesServiceCollectionExtensions
             return services
                 .RemoveService<IReadStoredQueries>()
                 .RemoveService<IWriteStoredQueries>()
-                .AddSingleton(sp => new RedisQueryStorage(databaseFactory(sp)))
+                .AddSingleton(sp => new RedisQueryStorage(databaseFactory(sp), queryExpiryTimeSpan))
```

Comment:
queryExpiration

Reply:
dunno why I ended up with the type in the name anyway